lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -205,6 +191,85 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private APIGatewayProxyResponseEvent getUserIdentityEndpointResponse(
+            String[] scopeClaims,
+            IpvSessionItem ipvSessionItem,
+            String userId,
+            AuditEventUser auditEventUser,
+            VerifiableCredential contraIndicatorsVc)
+            throws CiRetrievalException, VerifiableCredentialException, SqsException,
+                    HttpResponseExceptionWithErrorBody, CredentialParseException,
+                    UnrecognisedCiException {
+        if (!Arrays.asList(scopeClaims).contains(OPENID_SCOPE)) {
+            return getAccessDeniedApiGatewayProxyResponseEvent();
+        }
+
+        String ipvSessionId = ipvSessionItem.getIpvSessionId();
```

Comment:
out of curiosity, why do you have a preference for using `var` over an explicit type?

Reply:
That's a good question, and it's definitely a bit subjective, but for me the main reasons are:
- It clearly indicates a variable declaration and is more consistent with other languages
- It makes it easier to change/rename types later - e.g. recently we changed a bunch of code to pass `VerifiableCredential` rather than `SignedJWT`, and we had to change a bunch of intermediate variables that just added noise
- It's usually pretty obvious what the type is, either from context or by hovering over it in your IDE

Consider code like:
```
List<Credential> someCredentials = getSomeCredentials();
Credential specificCredential = findSpecificCredential(someCredentials);
doSomethingWith(specificCredential);
```
If you modify the name of the `Credential` type (or switch to a different class), you have to update these variable declarations, even though you aren't actually changing any logic in the method.